vendor/engines/production_credits/app/models/production_credits/venue.rb

```diff
@@ -9,7 +9,7 @@ def self.find_by_name_or_alias(name)
       if venue_name.empty?
         venue_name = name
       else
-        venue_name = venue_name.last.first 
+        venue_name = venue_name.last.first
```

Comment:
We didn't touch that line (other than Atom stripping the trailing space), so not sure what it's doing.  I expect it will disappear once we do venue aliases correctly.